src/core/layertree/qgslayertreemodellegendnode.h

```diff
@@ -189,6 +189,15 @@ class CORE_EXPORT QgsSymbolLegendNode : public QgsLayerTreeModelLegendNode
 {
     Q_OBJECT
 
+#ifdef SIP_RUN
```

Comment:
The important part is that it's in the sip file actually, this is just "template code". Either wait for travis to give you the complete version or have a look at an example (e.g. python/gui/auto_generated/editorwidgets/qgsspinbox.sip.in)

Reply:
travis gave it to me and I added it in the commit below.